v0/destinations/marketo_bulk_upload/fileUpload.js

```diff
@@ -304,21 +350,23 @@ const getImportID = async (input, config) => {
 
 const responseHandler = async (input, config) => {
   /**
-  * {
-  "importId" : <some-id>,
-  "pollURL" : <some-url-to-poll-status>,
-}
+  {
+    "importId" : <some-id>,
+    "pollURL" : <some-url-to-poll-status>,
+  }
   */
+  const fieldSchemaNames = await fetchFieldSchema(config);
```

Comment:
Les make the access token call `await getAccessToken(config)` here once and use the access token by passing it as args

Reply:
getAccessToken() called once and token gets passed around.